### PR TITLE
feat(eslint-plugin): new `injection-token-description`-rule 

### LINKED ISSUE #2023: 🤖 - Linter rule for `InjectionToken`-s description
## Problem:

Sometimes logs of problems with DI are not so informative.
See these logs of the real-problem:
```sh
NullInjectorError: StaticInjectorError(DynamicTestModule)
[TuiFilterByInputPipe -> InjectionToken A component that can be focused]: 
  StaticInjectorError(Platform: core)
    [TuiFilterByInputPipe -> InjectionToken A component that can be focused]: 
  NullInjectorError: No provider for InjectionToken A component that can be focused!
```
These logs tell that the pipe did not find this DI-token:
https://github.com/Tinkoff/taiga-ui/blob/96060358bf1d1008dbfa9c4566b1e5f1a02a9e7d/projects/cdk/tokens/focusable-item-accessor.ts#L4-L5

The description of this token is not so informative.
It can be easier to debug if the description contains the token's name.
For example:
```ts
export const TUI_FOCUSABLE_ITEM_ACCESSOR =
    new InjectionToken<TuiFocusableElementAccessor>(
        '[TUI_FOCUSABLE_ITEM_ACCESSOR] A component that can be focused',
    );
```

## Proposed solution
Write eslint-rule (with auto-fix) for description of `InjectionToken`.

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [X] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #2023

## What is the new behavior?
eslint-rule-demo

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -90,5 +90,6 @@ module.exports = {
                 case: 'kebabCase',
             },
         ],
+        'tui-rules/invalid-injection-token-description': 'error',
```
Comment (nsbarsukov) [inline .eslintrc.js:93]: Create new not-publishable (not-yet) library `@taiga-ui/eslint-plugin`

File: scripts/eslint/custom-rules/di-token-description.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                if (tokenDescription && !tokenDescription.includes(tokenName)) {
+                    context.report({
+                        node: tokenDescriptionNode,
+                        messageId: MESSAGE_ID,
+                        fix: fixer => {
+                            const [start, end] = tokenDescriptionNode.range;
+
+                            return fixer.insertTextBeforeRange(
+                                [start + 1, end],
+                                `[${tokenName}] `,
```
Comment (nsbarsukov) [inline scripts/eslint/custom-rules/di-token-description.js:35]:
```suggestion
                                `[${tokenName}]: `,
```

Comment (nsbarsukov) [reply scripts/eslint/custom-rules/di-token-description.js]: ✅

File: .eslintrc.js (hunk 1 above)
Comment (nsbarsukov) [reply .eslintrc.js]: ✅

**Review by splincode — APPROVED** [review]

**Pull request merged**